tex/set-theory/ordinal.tex

```diff
@@ -354,7 +394,14 @@ \section{Ordinal arithmetic}
 	(Look at $\gamma \times \beta \times \alpha$.)
 \end{exercise}
 
-Exponentiation can also be so defined, though the explicit construction is less natural.
+Similar to ordinal addition, defining $\alpha \cdot (\beta+1) = (\alpha \cdot \beta) + \alpha$ makes
+sure that if $\beta < \gamma$ then $\alpha \cdot \beta < \alpha \cdot \gamma$ --- as long as $\alpha
+> 0$.
+
+Exponentiation can also be so defined, though the explicit construction is less natural ---
+since we will not use this definition in the rest of the book, you may ignore it.
```

Comment:
should we really suggest the reader to ignore it?

On the one hand, it's indeed the natural generalization of ordinal addition and multiplication; on the other hand, it's not clear why we would care about it.

Reply:
I think it's fine to suggest ignoring something if we won't use it and it's distracting.